src/clic-tty.ads

```diff
@@ -50,10 +54,10 @@ is
        (if Color_Enabled and then Is_TTY then Text else Fallback);
    --  Intended to have a rich text and a safe alternative
 
-   function Info (Text : String := "") return String;
+   function Info (Text : String := "") return UTF_8_String;
```

Comment:
Why is the input not UTF_8_String as well?

Reply:
Right.